driver-core/src/test/java/com/datastax/driver/core/SchemaChangesTest.java

```diff
@@ -500,8 +501,8 @@ public void should_notify_of_keyspace_drop(String keyspace) throws InterruptedEx
         refreshSchema();
         for (Metadata m : metadatas()) {
             assertThat(m.getKeyspace(keyspace)).isNull();
-            assertThat(m.getReplicas(keyspace, Bytes.fromHexString("0xCAFEBABE"))).isEmpty();
         }
+        replicasPresentForKeyspace(keyspace).becomesFalse();
```

Comment:
The condition checks that the keyspace is present in the token map of all Clusters. So the negative would be "the keyspace is absent for at least one Cluster". Is it really what we want, or rather "the keyspace is absent for *all* Clusters"?